src/Microsoft.Restier.Core/Query/QueryExpressionContext.cs

```diff
@@ -174,6 +174,9 @@ private QueryModelReference ComputeModelReference()
             QueryModelReference modelReference = null;
 
             var methodCall = this.VisitedNode as MethodCallExpression;
+            var parameter = this.VisitedNode as ParameterExpression;
+            var member = this.VisitedNode as MemberExpression;
```

Comment:
Personally I feel the previous pattern is more efficient...You don't need to perform three casts at first.